packages/pricing-tier-list/partials/pricing-tier-item/index.tsx

```diff
@@ -0,0 +1,50 @@
+import Link from 'next/link'
+import classNames from 'classnames'
+import type { PricingTierItemProps } from '../../types'
+import s from './style.module.css'
+import Button from '@hashicorp/react-button'
+
+export default function PricingTierItem({
+  icon,
+  title,
+  label,
+  price,
+  consumption,
+  description,
+  cta,
+  supplementaryInfo,
+  size = 'lg',
+}: PricingTierItemProps): React.ReactElement {
+  return (
+    <div className={classNames(s.pricingTierItem, s[`${size}Size`])}>
+      {icon && <div className={s.icon}>{icon}</div>}
+      <h3 className={s.tierName}>{title}</h3>
+      <div className={s.details}>
+        {label && <span className={s.label}>{label}</span>}
+        {price && <span className={s.price}>{price}</span>}
+        {consumption && <span className={s.consumption}>{consumption}</span>}
+      </div>
+      <div
+        className={s.description}
+        dangerouslySetInnerHTML={{ __html: description }}
```

Comment:
Do we need to use dangerouslySetInnerHTML here to support the fields coming from Dato as HTML?

Reply:
Yeah exactly